src/courseware/course/sequence/Unit/UnitSuspense.jsx

```diff
@@ -24,19 +25,24 @@ const UnitSuspense = ({
     meta.contentTypeGatingEnabled && unit.containsContentTypeGatedContent
   );
 
-  const suspenseComponent = (message, Component) => (
-    <Suspense fallback={<PageLoading srMessage={formatMessage(message)} />}>
-      <Component courseId={courseId} />
-    </Suspense>
-  );
-
   return (
     <>
       {shouldDisplayContentGating && (
-        suspenseComponent(messages.loadingLockedContent, LockPaywall)
+        <Suspense fallback={<PageLoading srMessage={formatMessage(messages.loadingLockedContent)} />}>
+          <PluginSlot
+            id="fbe_message_plugin"
```

Comment:
Following the new naming convention as outlined above:

```suggestion
            id="fbe_message_slot"
```

Also, what does "fbe" stand for?  Might be worth making the slot ID a little longer if it means it's easier to deduce what it's for just from the name.

Reply:
It's short for feature based enrollment, I went back and named this 'gated content' since that's a bit more general purpose anyway.